plugins/MauticCrmBundle/Integration/SalesforceIntegration.php

```diff
@@ -827,4 +838,16 @@ public function getLeadData(\DateTime $startDate = null, \DateTime $endDate = nu
 
         return $leadActivity;
     }
+
+    /**
+     * @return FilesystemAdapter
+     *
+     * @TODO This should probably be moved to a service and system-wide
+     */
+    public function getCache()
+    {
+        $cacheDir = $this->dispatcher->getContainer()->getParameter('kernel.cache_dir');
```

Comment:
Please change this line to:

`$this->dispatcher->getContainer()->get('mautic.helper.paths')->getSystemPath('cache')`